src/nwbinspector/_inspection_cli.py

```diff
@@ -18,7 +18,17 @@
     print_to_console,
```

Comment:
The point of using the privatized submodules is that we can change them at will without affecting the public exposure - as such, no deprecation or changelog is needed since we 'do not support imports from such places'